### PR TITLE
Replaces composer with drush make + custom autoloader.

### PR DESCRIPTION
**Pull request by drkloc**
Issue #CIVIC-1951

### Acceptance Criteria

- [ ] run drush make against open_data_schema_map.make
- [ ] change any existing endpoint to use xml as output
- [ ] Save and test the endpoint

### CODE REVIEW

File: modules/open_data_schema_map_xml_output/autoload/autoload_classmap.php [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<?php
+
+$library_path=implode(
```
Comment (drkloc) [inline modules/open_data_schema_map_xml_output/autoload/autoload_classmap.php]: replace this with `libraries_get_path` call prior to merging

File: modules/open_data_schema_map_xml_output/autoload/autoload_namespaces.php [hunk 2]
```diff
@@ -0,0 +1,16 @@
+<?php
+
+$library_path=implode(
```
Comment (drkloc) [inline modules/open_data_schema_map_xml_output/autoload/autoload_namespaces.php]: replace this with `libraries_get_path` call prior to merging

File: modules/open_data_schema_map_xml_output/autoload/autoload_psr4.php [hunk 3]
```diff
@@ -0,0 +1,15 @@
+<?php
+
+$library_path=implode(
```
Comment (drkloc) [inline modules/open_data_schema_map_xml_output/autoload/autoload_psr4.php]: replace this with `libraries_get_path` call prior to merging

**Comment by dafeder** [comment]
@jacintocapote @drkloc This PR is superseded by #57 right? Can one of you confirm and close?

**Pull request closed without merging**